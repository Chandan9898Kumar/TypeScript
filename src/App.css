body {
  margin: 0;
  top: 0;
  box-sizing: border-box;
}

h1 {
  text-align: center;
  text-transform: capitalize;
  font-family: "Luckiest Guy", cursive;
  letter-spacing: 2px;
  text-shadow: 2px 5px 5px rgb(227 47 47 / 40%);
  margin: 20px auto;
}

.customH1Tag {
  text-transform: var(--text-transform);
  color: var(--head-color);
}

img {
  width: 500px;
  height: 300px;
  aspect-ratio: 1 / 1;
  object-fit: contain;
  background: aliceblue;
  border-radius: 20px;
}

.image-carousel {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: row;
  flex-wrap: wrap;
}

.image-button {
  cursor: pointer;
  cursor: pointer;
  font-size: 20px;
  color: var(--text-color);
  font-weight: 500;
  padding: 10px 10px;
}

.customTag {
  margin: 0px;
  /* padding: 10px 10px; */
}
