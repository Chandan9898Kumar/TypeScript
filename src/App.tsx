import { motion } from "framer-motion";
import { lazy, Suspense,useState } from "react";
import "./App.css";
import Loader from "./Components/Loader/Loader";
const Carousel = lazy(() => import("./Pages/Carousel/Carousel"));
const Header = lazy(() => import("./Components/Header/Header"));
const TrafficLight = lazy(() => import("./Pages/TrafficLight/TrafficLight"));
const PausableCounter = lazy(() => import("./Pages/PausableCounter/PausableCounter"));
const AccordionContainer = lazy(() => import("./Pages/Accordion/Accordion"));
const StartRating = lazy(() => import("./Pages/StartRating/StarRating"));
const Toggle = lazy(() => import("./Components/Toggle/Toggle"));
const MessageComponent = lazy(() => import("./Pages/Message/Message"));
const Stepper = lazy(() => import("./Pages/Stepper/Stepper"));
const AutoSuggestion = lazy(()=>import('./Pages/AutoSuggestion/AutoSuggestion'))
const InputFocusPartTwo = lazy(()=>import('./Pages/OTP/OtpFieldTwo'))
const ScrollOnElement = lazy(()=>import('./Components/InfiniteScroll/ScrollerOnElement'))
const InfiniteScroller = lazy(()=>import('./Pages/InfiniteScroller/InfiniteScroller'))
const FeatureFlagPage = lazy(()=>import('./Pages/FeatureFlagPage/FeatureFlagPage'))
const StopWatch = lazy(()=>import('./Pages/StopWatch/StopWatch'))
const TicTacToe = lazy(()=>import('./Pages/TicTaeToe/TicTacToe'))
const FindRingsAndRodCombination = lazy(()=>import('./Pages/RingsAndRods/RingsRods'))
const MainProgress = lazy(() => import("./Pages/ProgressBarNTimes/MainProgress"));
const FilterTable = lazy(()=>import('./Pages/FilterTable/FilterTable'))

const App: React.FC = () => {
  const [scrollData, setScrollData] = useState<number>(50);
  return (
    <>
      <Suspense fallback={<Loader />}>
        <header>
          <motion.div
            style={{
              display: "flex",
              justifyContent: "space-between",
              alignItems: "center",
              padding:"10px 10px"
            }}
            initial={{ opacity: 0, y: -20 }}
            animate={{ opacity: 1, y: 0 }}
            transition={{
              duration: 0.7,
              ease: [0.6, -0.05, 0.01, 0.99],
            }}
          >
            <Header title="Machine Coding In TypeScript..." className="customTag" />
            <Toggle />
          </motion.div>
        </header>{" "}

        <FilterTable />
        <MainProgress />
        <FindRingsAndRodCombination />
        <TicTacToe />
        <StopWatch />
        <FeatureFlagPage />
        <InputFocusPartTwo />
        <Carousel />
        <TrafficLight />
        <PausableCounter />
        <AccordionContainer />
        <StartRating />
        <MessageComponent />
        <Stepper />
        <AutoSuggestion />
        <ScrollOnElement scrollData={scrollData}  setScrollData={setScrollData}/>
        <InfiniteScroller />
       
      </Suspense>
    </>
  );
};

export default App;
