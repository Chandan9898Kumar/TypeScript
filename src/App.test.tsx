import { render, screen } from '@testing-library/react';
import App from './App';
import '@testing-library/jest-dom';

jest.mock('./Pages/Carousel/Carousel', () => () => <div>Carousel Component</div>);
jest.mock('./Components/Header/Header', () => ({ title, className }: { title: string, className: string }) => (
  <div className={className}>{title}</div>
));
jest.mock('./Components/Toggle/Toggle', () => () => <div>Toggle Component</div>);
// ... mock other lazy components similarly

describe('App Component', () => {
  it('renders without crashing', () => {
    render(<App />);
    expect(screen.getByTestId('loader')).toBeInTheDocument();
  });

  it('renders header with correct title', async () => {
    render(<App />);
    const headerText = await screen.findByText('Machine Coding In TypeScript...');
    expect(headerText).toBeInTheDocument();
  });

  it('renders with correct header className', async () => {
    render(<App />);
    const headerElement = await screen.findByText('Machine Coding In TypeScript...');
    expect(headerElement).toHaveClass('customTag');
  });

  it('passes correct scroll data props to ScrollOnElement', () => {
    const { container } = render(<App />);
    // Initial scroll data should be 50
    expect(container).toMatchSnapshot();
  });

  it('renders all lazy-loaded components', async () => {
    render(<App />);
    
    // Wait for components to load
    const carousel = await screen.findByText('Carousel Component');
    const toggle = await screen.findByText('Toggle Component');
    
    expect(carousel).toBeInTheDocument();
    expect(toggle).toBeInTheDocument();
  });
});

