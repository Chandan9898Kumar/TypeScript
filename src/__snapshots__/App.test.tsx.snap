// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component passes correct scroll data props to ScrollOnElement 1`] = `
<div>
  <header>
    <div
      style="display: flex; justify-content: space-between; align-items: center; padding: 10px 10px; opacity: 0; transform: translateY(-20px);"
    >
      <div
        class="customTag"
      >
        Machine Coding In TypeScript...
      </div>
      <div>
        Toggle Component
      </div>
    </div>
  </header>
   
  <div>
    <h1>
      OTP Field
    </h1>
    <div
      class="mainTwo"
    >
      <div
        class="main"
      >
        <input
          maxlength="1"
          type="text"
          value=""
        />
      </div>
      <div
        class="main"
      >
        <input
          maxlength="1"
          type="text"
          value=""
        />
      </div>
      <div
        class="main"
      >
        <input
          maxlength="1"
          type="text"
          value=""
        />
      </div>
      <div
        class="main"
      >
        <input
          maxlength="1"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
  <div>
    Carousel Component
  </div>
  <section>
    <div
      class="customH1Tag"
    >
      Traffic Signal 
    </div>
    <div
      class="main"
    >
      <div
        class="signal"
      >
        <div
          class="red"
          style="background-color: red;"
        />
        <div
          class="yellow"
        />
        <div
          class="green"
        />
      </div>
    </div>
  </section>
  <section>
    <div
      class="customH1Tag"
    >
      Pausable Counter
    </div>
    <div
      class="container"
    >
      <div
        class="buttons"
        style="opacity: 0; transform: translateY(20px);"
      >
        <button
          tabindex="0"
        >
          Start
        </button>
        <button
          disabled=""
          tabindex="0"
        >
          Pause
        </button>
        <button
          disabled=""
          tabindex="0"
        >
          Reset
        </button>
      </div>
      <span
        class="text"
        style="opacity: 0; transform: translateX(20px) scale(0.9);"
      >
        <span
          style="color: rgb(107, 114, 128); opacity: 0.7;"
        >
          Count
           
          :
           
        </span>
        <span>
          0
        </span>
      </span>
    </div>
  </section>
  <section>
    <div
      class="customH1Tag"
    >
      Accordion
    </div>
    <ul
      aria-label="Movie descriptions accordion"
      class="container"
      role="button"
    >
      <li
        class="lists"
        data-items="{"id":1,"title":"The 400 Blows","description":"A young boy, left without attention, delves into a life of petty crime.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-1"
          >
            The 400 Blows
          </span>
        </span>
      </li>
      <li
        class="lists"
        data-items="{"id":2,"title":"La haine","description":"24 hours in the lives of three young men in the French suburbs the day after a violent riot.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-2"
          >
            La haine
          </span>
        </span>
      </li>
      <li
        class="lists"
        data-items="{"id":3,"title":"The Godfather","description":"The aging patriarch of an organized crime dynasty transfers control of his clandestine empire to his reluctant son.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-3"
          >
            The Godfather
          </span>
        </span>
      </li>
      <li
        class="lists"
        data-items="{"id":4,"title":"The Godfather Part II","description":"The early life and career of Vito Corleone in 1920s New York City is portrayed, while his son, Michael, expands and tightens his grip on the family crime syndicate.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-4"
          >
            The Godfather Part II
          </span>
        </span>
      </li>
      <li
        class="lists"
        data-items="{"id":5,"title":" Man Bites Dog","description":"A film crew follows a ruthless thief and heartless killer as he goes about his daily routine. But complications set in when the film crew lose their objectivity and begin lending a hand.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-5"
          >
             Man Bites Dog
          </span>
        </span>
      </li>
      <li
        class="lists"
        data-items="{"id":6,"title":"The Departed","description":"An undercover cop and a mole in the police attempt to identify each other while infiltrating an Irish gang in South Boston.","isOpen":false}"
        role="list"
      >
        <span
          class="accordionHeader"
        >
          <svg
            class="bi bi-arrow-right"
            fill="currentColor"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
              fill-rule="evenodd"
            />
          </svg>
            
          <span
            id="title-6"
          >
            The Departed
          </span>
        </span>
      </li>
    </ul>
  </section>
  <section>
    <div
      class="customH1Tag"
    >
      Start Rating
    </div>
    <div
      aria-label="Star rating"
      class="container"
      role="group"
    >
      <div
        aria-pressed="false"
        class="stars"
        role="button"
        style="opacity: 0; transform: scale(0.5);"
        tabindex="0"
      >
        <div>
          <svg
            aria-hidden="true"
            class="bi bi-star"
            fill="cornflowerblue"
            height="45"
            viewBox="0 0 16 16"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.866 14.85c-.078.444.36.791.746.593l4.39-2.256 4.389 2.256c.386.198.824-.149.746-.592l-.83-4.73 3.522-3.356c.33-.314.16-.888-.282-.95l-4.898-.696L8.465.792a.513.513 0 0 0-.927 0L5.354 5.12l-4.898.696c-.441.062-.612.636-.283.95l3.523 3.356-.83 4.73zm4.905-2.767-3.686 1.894.694-3.957a.56.56 0 0 0-.163-.505L1.71 6.745l4.052-.576a.53.53 0 0 0 .393-.288L8 2.223l1.847 3.658a.53.53 0 0 0 .393.288l4.052.575-2.906 2.77a.56.56 0 0 0-.163.506l.694 3.957-3.686-1.894a.5.5 0 0 0-.461 0z"
            />
          </svg>
        </div>
      </div>
      <div
        aria-pressed="false"
        class="stars"
        role="button"
        style="opacity: 0; transform: scale(0.5);"
        tabindex="0"
      >
        <div>
          <svg
            aria-hidden="true"
            class="bi bi-star"
            fill="cornflowerblue"
            height="45"
            viewBox="0 0 16 16"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.866 14.85c-.078.444.36.791.746.593l4.39-2.256 4.389 2.256c.386.198.824-.149.746-.592l-.83-4.73 3.522-3.356c.33-.314.16-.888-.282-.95l-4.898-.696L8.465.792a.513.513 0 0 0-.927 0L5.354 5.12l-4.898.696c-.441.062-.612.636-.283.95l3.523 3.356-.83 4.73zm4.905-2.767-3.686 1.894.694-3.957a.56.56 0 0 0-.163-.505L1.71 6.745l4.052-.576a.53.53 0 0 0 .393-.288L8 2.223l1.847 3.658a.53.53 0 0 0 .393.288l4.052.575-2.906 2.77a.56.56 0 0 0-.163.506l.694 3.957-3.686-1.894a.5.5 0 0 0-.461 0z"
            />
          </svg>
        </div>
      </div>
      <div
        aria-pressed="false"
        class="stars"
        role="button"
        style="opacity: 0; transform: scale(0.5);"
        tabindex="0"
      >
        <div>
          <svg
            aria-hidden="true"
            class="bi bi-star"
            fill="cornflowerblue"
            height="45"
            viewBox="0 0 16 16"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.866 14.85c-.078.444.36.791.746.593l4.39-2.256 4.389 2.256c.386.198.824-.149.746-.592l-.83-4.73 3.522-3.356c.33-.314.16-.888-.282-.95l-4.898-.696L8.465.792a.513.513 0 0 0-.927 0L5.354 5.12l-4.898.696c-.441.062-.612.636-.283.95l3.523 3.356-.83 4.73zm4.905-2.767-3.686 1.894.694-3.957a.56.56 0 0 0-.163-.505L1.71 6.745l4.052-.576a.53.53 0 0 0 .393-.288L8 2.223l1.847 3.658a.53.53 0 0 0 .393.288l4.052.575-2.906 2.77a.56.56 0 0 0-.163.506l.694 3.957-3.686-1.894a.5.5 0 0 0-.461 0z"
            />
          </svg>
        </div>
      </div>
      <div
        aria-pressed="false"
        class="stars"
        role="button"
        style="opacity: 0; transform: scale(0.5);"
        tabindex="0"
      >
        <div>
          <svg
            aria-hidden="true"
            class="bi bi-star"
            fill="cornflowerblue"
            height="45"
            viewBox="0 0 16 16"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.866 14.85c-.078.444.36.791.746.593l4.39-2.256 4.389 2.256c.386.198.824-.149.746-.592l-.83-4.73 3.522-3.356c.33-.314.16-.888-.282-.95l-4.898-.696L8.465.792a.513.513 0 0 0-.927 0L5.354 5.12l-4.898.696c-.441.062-.612.636-.283.95l3.523 3.356-.83 4.73zm4.905-2.767-3.686 1.894.694-3.957a.56.56 0 0 0-.163-.505L1.71 6.745l4.052-.576a.53.53 0 0 0 .393-.288L8 2.223l1.847 3.658a.53.53 0 0 0 .393.288l4.052.575-2.906 2.77a.56.56 0 0 0-.163.506l.694 3.957-3.686-1.894a.5.5 0 0 0-.461 0z"
            />
          </svg>
        </div>
      </div>
      <div
        aria-pressed="false"
        class="stars"
        role="button"
        style="opacity: 0; transform: scale(0.5);"
        tabindex="0"
      >
        <div>
          <svg
            aria-hidden="true"
            class="bi bi-star"
            fill="cornflowerblue"
            height="45"
            viewBox="0 0 16 16"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.866 14.85c-.078.444.36.791.746.593l4.39-2.256 4.389 2.256c.386.198.824-.149.746-.592l-.83-4.73 3.522-3.356c.33-.314.16-.888-.282-.95l-4.898-.696L8.465.792a.513.513 0 0 0-.927 0L5.354 5.12l-4.898.696c-.441.062-.612.636-.283.95l3.523 3.356-.83 4.73zm4.905-2.767-3.686 1.894.694-3.957a.56.56 0 0 0-.163-.505L1.71 6.745l4.052-.576a.53.53 0 0 0 .393-.288L8 2.223l1.847 3.658a.53.53 0 0 0 .393.288l4.052.575-2.906 2.77a.56.56 0 0 0-.163.506l.694 3.957-3.686-1.894a.5.5 0 0 0-.461 0z"
            />
          </svg>
        </div>
      </div>
    </div>
  </section>
  <section
    class="main"
  >
    <h1>
      Custom Modal Component 
    </h1>
    <header
      class="headers"
    >
      ▶️ A Modal is a dialog that appears on top of the app's content, and must be dismissed by the app before interaction can resume. It is useful as a select component when there are a lot of options to choose from, or when filtering items in a list, as well as many other use cases.
    </header>
    <div
      class="btn"
    >
      <button
        aria-label="open modal"
        type="button"
      >
        Open
      </button>
    </div>
  </section>
  <section
    class="container"
  >
    <div
      class="stepper"
    >
      <div
        class="main"
      >
        <div
          class="ids"
          data-item="true"
        >
          <span>
            1
          </span>
        </div>
        <div
          class="item"
        >
          first Step
        </div>
      </div>
      <div
        class="main"
      >
        <div
          class="ids"
          data-item="false"
        >
          <span>
            2
          </span>
        </div>
        <div
          class="item"
        >
          second Step
        </div>
      </div>
      <div
        class="main"
      >
        <div
          class="ids"
          data-item="false"
        >
          <span>
            3
          </span>
        </div>
        <div
          class="item"
        >
          third Step
        </div>
      </div>
      <div
        class="main"
      >
        <div
          class="ids"
          data-item="false"
        >
          <span>
            4
          </span>
        </div>
        <div
          class="item"
        >
          fourth Step
        </div>
      </div>
      <div
        class="main"
      >
        <div
          class="ids"
          data-item="false"
        >
          <span>
            5
          </span>
        </div>
        <div
          class="item"
        >
          fifth Step
        </div>
      </div>
      <div
        class="bar"
      >
        <div
          class="progress"
          style="transform: scaleX(0); transition: transform 0.3s ease-out; transform-origin: left; display: block; width: 100%;"
        />
      </div>
    </div>
    <div
      class="products"
    >
      <div
        class="wrapper"
      >
        <div
          class="clash-card barbarian"
        >
          <div
            class="clash-card__image clash-card__image--barbarian"
          >
            <img
              alt="barbarian"
              loading="lazy"
              src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/195612/barbarian.png"
            />
          </div>
          <div
            class="centerItem"
          >
            <div
              class="clash-card__level clash-card__level--barbarian"
            >
              Level 1
            </div>
            <div
              class="clash-card__unit-name"
            >
              The Barbarian
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="btns"
    >
      <button
        aria-label="back button"
        disabled=""
        type="button"
      >
        Back
      </button>
      <button
        aria-label="next button"
        type="button"
      >
        Next
      </button>
    </div>
  </section>
  <div>
    fetch is not defined
  </div>
  <div
    class="main"
  >
    <h1>
      This is Infinite Scroller On Element
    </h1>
    <div
      class="scroller"
    >
      <p>
        1
      </p>
      <p>
        2
      </p>
      <p>
        3
      </p>
      <p>
        4
      </p>
      <p>
        5
      </p>
      <p>
        6
      </p>
      <p>
        7
      </p>
      <p>
        8
      </p>
      <p>
        9
      </p>
      <p>
        10
      </p>
      <p>
        11
      </p>
      <p>
        12
      </p>
      <p>
        13
      </p>
      <p>
        14
      </p>
      <p>
        15
      </p>
      <p>
        16
      </p>
      <p>
        17
      </p>
      <p>
        18
      </p>
      <p>
        19
      </p>
      <p>
        20
      </p>
      <p>
        21
      </p>
      <p>
        22
      </p>
      <p>
        23
      </p>
      <p>
        24
      </p>
      <p>
        25
      </p>
      <p>
        26
      </p>
      <p>
        27
      </p>
      <p>
        28
      </p>
      <p>
        29
      </p>
      <p>
        30
      </p>
      <p>
        31
      </p>
      <p>
        32
      </p>
      <p>
        33
      </p>
      <p>
        34
      </p>
      <p>
        35
      </p>
      <p>
        36
      </p>
      <p>
        37
      </p>
      <p>
        38
      </p>
      <p>
        39
      </p>
      <p>
        40
      </p>
      <p>
        41
      </p>
      <p>
        42
      </p>
      <p>
        43
      </p>
      <p>
        44
      </p>
      <p>
        45
      </p>
      <p>
        46
      </p>
      <p>
        47
      </p>
      <p>
        48
      </p>
      <p>
        49
      </p>
      <p>
        50
      </p>
    </div>
  </div>
</div>
`;
